dev-infrastructure/docs/development-setup.md

```diff
@@ -122,6 +122,12 @@ This will create:
 1. A custom role definition with required access as defined in `dev-infrastructure/scripts/mock-dev-role-definition.json`
 1. A service principal/application using the created cert as its authentication, and given access based on the custom role definition
 
+In the newly created Resource Group (test-RG):
```

Comment:
For the other resources we don't specify test values in the documentation like test-RG, ... . We should be consistent with the guide and remove references. to example values. The same applies to test-Vnet, test-NSG, ...